src/SignUpIn/DelivererGroup/DelivererGroupSignUp2.js

```diff
@@ -11,16 +11,16 @@ class DelivererGroupSignUp2 extends Component {
                 <div className="signup-content">
                     <div className="form-block">
                         <label className="form-component">Account Details</label><br />
-                        <input name="email" type="text" className="form-component" placeholder="Email" defaultValue={this.props.fieldValues.email} /><br />
-                        <input name="password" type="password" className="form-component" placeholder="Password" defaultValue={this.props.fieldValues.password} />
-                        <input type="password" className="form-component" placeholder="Confirm Password" defaultValue={this.props.fieldValues.password} />
+                        <input name="email" type="text" className="form-component" placeholder="Email" defaultValue={this.props.fieldValues.email} required /><br />
+                        <input name="password" type="password" className="form-component" placeholder="Password" defaultValue={this.props.fieldValues.password} required />
+                        <input type="password" className="form-component" placeholder="Confirm Password" defaultValue={this.props.fieldValues.password} required />
                     </div>
                     <div className="form-block">
                         <label className="form-component">Coordinator Contact Information</label><br />
-                        <input name="contactName" type="text" className="form-component" placeholder="Name" defaultValue={this.props.fieldValues.contactName} /><br />
-                        <input name="contactPosition" type="text" className="form-component" placeholder="Position" defaultValue={this.props.fieldValues.contactPosition} /><br />
-                        <input name="contactEmail" type="text" className="form-component" placeholder="Email" defaultValue={this.props.fieldValues.contactEmail} /><br />
-                        <input name="contactNumber" type="text" className="form-component" placeholder="Phone" defaultValue={this.props.fieldValues.contactNumber} /><br />
+                        <input name="contactName" type="text" className="form-component" placeholder="Name" defaultValue={this.props.fieldValues.contactName} required /><br />
+                        <input name="contactPosition" type="text" className="form-component" placeholder="Position" defaultValue={this.props.fieldValues.contactPosition} required /><br />
+                        <input name="contactEmail" type="text" className="form-component" placeholder="Email" defaultValue={this.props.fieldValues.contactEmail} required /><br />
+                        <input name="contactNumber" type="text" className="form-component" placeholder="Phone" defaultValue={this.props.fieldValues.contactNumber} required /><br />
```

Comment:
might be a good idea to change all phone number inputs to have these attributes (validates phone number format) 
```type="tel" pattern="[0-9]{3}-[0-9]{3}-[0-9]{4}"```